syntax/dirvish.vim

```diff
@@ -7,4 +7,6 @@ let s:sep = exists('+shellslash') && !&shellslash ? '\\' : '\/'
 exe 'syntax match DirvishPathHead ''\v.*'.s:sep.'\ze[^'.s:sep.']+'.s:sep.'?$'' conceal'
 exe 'syntax match DirvishPathTail ''\v[^'.s:sep.']+'.s:sep.'$'''
 
+hi def link DirvishPathTail Directory
```

Comment:
Good call with "default"! That actually accomplishes the purported goal, unlike the old way :)

I changed the commit to fully spell out the commands.